src/base/bittorrent/session.cpp

```diff
@@ -138,13 +145,27 @@ namespace
         return result;
     }
 
+#if LIBTORRENT_VERSION_NUM < 10200
+    // we declare this shortcut here, because...
+    inline QString fromStdString(const std::string& str)
+    {
+        return QString::fromStdString(str);
+    }
+#else
+    // ... if libtorrent uses string_view, we need the other overload only in this file.
+    inline QString fromStdString(const libtorrent::string_view& str)
```

Comment:
I don't think there's a good reason to take the `string_view` by const ref here. it's already a reference type